src/eos-crash-metrics.c

```diff
@@ -137,6 +182,122 @@ normalize_path (char *path)
     path[i] = path[i] == '!' ? '/' : path[i];
 }
 
+static inline void
+sanitize_name (gchar *name)
+{
+  gchar *tmp = strchr (name, '/');
+
+  if (tmp != NULL)
+    *tmp = '\0';
+}
+
+static const char *
+get_associated_runtime (FlatpakInstalledRef *ref, GError **error)
+{
+  g_autoptr(GBytes) metadata = NULL;
+  gchar *runtime = NULL;
+  g_autoptr(GKeyFile) key_file = NULL;
+  const gchar *metadata_data;
+  gsize metadata_len;
+  g_autoptr(GError) local_error = NULL;
+
+  metadata = flatpak_installed_ref_load_metadata (FLATPAK_INSTALLED_REF (ref), NULL, &local_error);
+
+  if (metadata == NULL)
+    {
+      g_propagate_error (error, g_steal_pointer (&local_error));
```

Comment:
Since you don't do anything with the error locally, it looks like you could simply pass `error` to the above function

Reply:
Yes, worked.